src/tests/OutputPresenter.test.ts

```diff
@@ -0,0 +1,94 @@
+import { OutputPresenter } from "../OutputPresenter";
+import { templates } from "../Template";
+describe("output presenter", () => {
+  let presenter: OutputPresenter;
+  let mood = templates[0];
+
+  window.HTMLMediaElement.prototype.play = jest.fn();
+  window.HTMLMediaElement.prototype.pause = jest.fn();
+  window.HTMLMediaElement.prototype.load = jest.fn();
+  beforeEach(() => {
+    presenter = new OutputPresenter();
+  });
+
+  describe("playVideo", () => {
+    it("should set play viriable as true when played second is greater than total duration", () => {
+      presenter.overallPlayedSeconds = 10;
+      presenter.currLength.totalDuration = 5;
+      presenter.music = new Audio(mood.tracks[0].musicTrack);
+      presenter.playVideo();
+      expect(presenter.music.play).toHaveBeenCalled();
+      expect(presenter.play).toBe(true);
+    });
+
+    it("should set play as true", () => {
```

Comment:
`should play the video`
This should also be the first test in this series, as it's the most obvious thing that should happen in `playVideo`